From c82b822598e3a8274c0582615bea75b66aa740b2 Mon Sep 17 00:00:00 2001
From: Herman van Hazendonk <github.com@herrie.org>
Date: Tue, 29 Nov 2022 15:35:41 +0100
Subject: [PATCH] com.palm.db.role.json.in: More generic app access

For our purposes it's easier to just whitelist all the com.webos, com.palm and com.webosports apps here.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
Upstream-Status: Pending
---
 files/sysbus/dynamic/com.palm.db.role.json.in | 12 ++++--------
 files/sysbus/static/com.palm.db.role.json.in  | 12 ++++--------
 2 files changed, 8 insertions(+), 16 deletions(-)

diff --git a/files/sysbus/dynamic/com.palm.db.role.json.in b/files/sysbus/dynamic/com.palm.db.role.json.in
index 5b49438..c876a82 100644
--- a/files/sysbus/dynamic/com.palm.db.role.json.in
+++ b/files/sysbus/dynamic/com.palm.db.role.json.in
@@ -6,19 +6,15 @@
 "permissions": [
 {
 "service":"com.palm.db",
-"outbound":["com.palm.service.backup","com.palm.systemmanager","com.palm.activitymanager", "com.webos.service.attachedstoragemanager", "com.webos.settingsservice", "com.webos.service.devicereset","com.webos.service.micomservice","com.webos.service.faultmanager", "com.palm.db"]
-},
+"outbound":["com.webos.*", "com.palm.*", "org.webosports.*"]},
 {
 "service":"com.palm.tempdb",
-"outbound":["com.palm.activitymanager","com.palm.systemmanager", "com.webos.service.attachedstoragemanager", "com.webos.settingsservice", "com.webos.service.devicereset","com.webos.service.micomservice","com.webos.service.faultmanager", "com.palm.db", "com.palm.tempdb"]
-},
+"outbound":["com.webos.*", "com.palm.*", "org.webosports.*"]},
 {
 "service":"com.webos.mediadb",
-"outbound":["com.palm.activitymanager","com.palm.systemmanager", "com.webos.service.attachedstoragemanager", "com.webos.settingsservice", "com.webos.service.devicereset","com.webos.service.micomservice","com.webos.service.faultmanager", "com.palm.db", "com.webos.mediadb"]
-},
+"outbound":["com.webos.*", "com.palm.*", "org.webosports.*"]},
 {
 "service":"com.webos.epgdb",
-"outbound":["com.palm.activitymanager","com.palm.systemmanager", "com.webos.service.attachedstoragemanager", "com.webos.settingsservice","com.webos.service.devicereset","com.webos.service.micomservice","com.webos.service.faultmanager", "com.palm.db"]
-}
+"outbound":["com.webos.*", "com.palm.*", "org.webosports.*"]}
 ]
 }
\ No newline at end of file
diff --git a/files/sysbus/static/com.palm.db.role.json.in b/files/sysbus/static/com.palm.db.role.json.in
index 5b49438..c876a82 100644
--- a/files/sysbus/static/com.palm.db.role.json.in
+++ b/files/sysbus/static/com.palm.db.role.json.in
@@ -6,19 +6,15 @@
 "permissions": [
 {
 "service":"com.palm.db",
-"outbound":["com.palm.service.backup","com.palm.systemmanager","com.palm.activitymanager", "com.webos.service.attachedstoragemanager", "com.webos.settingsservice", "com.webos.service.devicereset","com.webos.service.micomservice","com.webos.service.faultmanager", "com.palm.db"]
-},
+"outbound":["com.webos.*", "com.palm.*", "org.webosports.*"]},
 {
 "service":"com.palm.tempdb",
-"outbound":["com.palm.activitymanager","com.palm.systemmanager", "com.webos.service.attachedstoragemanager", "com.webos.settingsservice", "com.webos.service.devicereset","com.webos.service.micomservice","com.webos.service.faultmanager", "com.palm.db", "com.palm.tempdb"]
-},
+"outbound":["com.webos.*", "com.palm.*", "org.webosports.*"]},
 {
 "service":"com.webos.mediadb",
-"outbound":["com.palm.activitymanager","com.palm.systemmanager", "com.webos.service.attachedstoragemanager", "com.webos.settingsservice", "com.webos.service.devicereset","com.webos.service.micomservice","com.webos.service.faultmanager", "com.palm.db", "com.webos.mediadb"]
-},
+"outbound":["com.webos.*", "com.palm.*", "org.webosports.*"]},
 {
 "service":"com.webos.epgdb",
-"outbound":["com.palm.activitymanager","com.palm.systemmanager", "com.webos.service.attachedstoragemanager", "com.webos.settingsservice","com.webos.service.devicereset","com.webos.service.micomservice","com.webos.service.faultmanager", "com.palm.db"]
-}
+"outbound":["com.webos.*", "com.palm.*", "org.webosports.*"]}
 ]
 }
\ No newline at end of file
-- 
2.37.3.windows.1

